rustls/src/msgs/handshake_test.rs

```diff
@@ -1201,6 +1201,6 @@ fn can_decode_server_hello_from_api_devicecheck_apple_com() {
 fn wrapped_dn_encoding() {
     let subject = b"subject";
     let dn = DistinguishedName::in_sequence(&subject[..]);
-    let expected_prefix = vec![ring::io::der::Tag::Sequence as u8, subject.len() as u8];
+    let expected_prefix = vec![0x30u8, subject.len() as u8];
```

Comment:
Ah, sorry for sneaking that in. Still, maybe this should be a `const DER_SEQUENCE` or whatever? IIRC you worked on the no magic constants lint that one time!